azure-pipelines/vs-insertion-experimental.yml

```diff
@@ -56,8 +71,29 @@ extends:
             artifactName: 'PackageArtifacts'
             targetPath: '$(Build.ArtifactStagingDirectory)/PackageArtifacts'
         steps:
+        # the CI build creates a sourcebuild intermediate package that is not signed, remove it to avoid warning from Guardian
+        - task: Powershell@2
+          name: RemoveSourceBuildPackage
+          displayName: Remove sourcebuild package
+          inputs:
+            targetType: inline
+            script: |
+              $folder = "$(Build.ArtifactStagingDirectory)/PackageArtifacts"
+              Get-ChildItem -Path $folder -Filter 'Microsoft.SourceBuild.Intermediate*.nupkg' | Remove-Item -Force
```

Comment:
Can you add an exclude pattern to the `PackageArtifacts` download? Something like `!**/Microsoft.SourceBuild.Intermediate*.nupkg`?